### ISSUE TITLE
get_iterator() freezes with new_tweets=false

### ISSUE THREAD

**Issue by reuning** [geduldig/TwitterAPI#119]
I'm having a problem with pulling tweets from a user using `get_iterator`. It seems to go threw all the old tweets and then pauses. I've set `new_tweets=False` but that doesn't seem to change the behavior. 

Below is a minimal working example of what is happening. It seems to freeze around ~920 tweets. Maybe I am just confused with how something is working? 

Thanks for any help.

```
usernames = ['maggieNYT', 'davidfrum']
for u in usernames:
    print(u)
    out = []
    pager = TwitterPager(api, 'statuses/user_timeline', 
                         {'screen_name':u, 'count':200, 'include_rts':False })
    count =0 
    for item in pager.get_iterator(wait=3.5, new_tweets=False):
        count += 1
        print(count)
```

**Comment by geduldig** [comment]
Hi - I'll post a fix in the next day or so. Thanks for posting!

**Comment by reuning** [comment]
FYI, I've been playing around and I think this is a problem with what the API is returning. There doesn't seem to be any cursor on the API return.

**Comment by reuning** [comment]
So I think I narrowed down where the problem is. In https://github.com/geduldig/TwitterAPI/commit/43aee51d5f47468e50c42a8fd7a7135fdd622da4#diff-b3849470e0f3f5e1d3445e061a917464L81 it deleted the check to see if ID was empty. I'm not sure how exactly to re-add that without messing things up previously. 

I rolled back to 2.5.7 and the bug is gone.

**Comment by geduldig** [comment]
Hold on. I have a couple of fixes with this one. I'll push probably today or tomorrow.

**Comment by geduldig** [comment]
This problem should be fixed in the latest version, 2.5.9. Please try it out.

**Comment by reuning** [comment]
Looks like it works perfectly now. Thanks.
